### PR TITLE
Add extra config to `Kaniko` docs

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/stacks-and-components/component-guide) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: docs/book/stacks-and-components/component-guide/image-builders/kaniko.md [hunk 1]
```diff
@@ -139,6 +140,7 @@ zenml image-builder register <NAME> \
     --kubernetes_context=<KUBERNETES_CONTEXT> \
     --volume_mounts='[{"name": "docker-config", "mountPath": "/kaniko/.docker/"}]' \
     --volumes='[{"name": "docker-config", "configMap": {"name": "docker-config"}}]'
+    # --executor_args='["--compressed-caching=false", "--use-new-run=true"]'
```
Comment (strickvl) [inline docs/book/stacks-and-components/component-guide/image-builders/kaniko.md:143]: why has this been commented out?

File: docs/book/stacks-and-components/component-guide/image-builders/kaniko.md [hunk 2]
```diff
@@ -153,8 +155,7 @@ information.
 
 #### Passing additional parameters to the Kaniko build
 
-If you want to pass [additional flags](https://github.com/GoogleContainerTools/kaniko#additional-flags) to the Kaniko
-build, pass them as a JSON string when registering your image builder in the stack:
+you can pass additional parameters to the Kaniko build by setting the `executor_args` attribute of the image builder.
```
Comment (strickvl) [inline docs/book/stacks-and-components/component-guide/image-builders/kaniko.md:158]:
```suggestion
You can pass additional parameters to the Kaniko build by setting the `executor_args` attribute of the image builder.
```

File: docs/book/stacks-and-components/component-guide/image-builders/kaniko.md (hunk 1 above)
Comment (strickvl) [reply docs/book/stacks-and-components/component-guide/image-builders/kaniko.md:143]: It's just there as an illustration?

**Review by strickvl — CHANGES_REQUESTED** [review]

File: docs/book/stacks-and-components/component-guide/image-builders/kaniko.md (hunk 1 above)
Comment (safoinme) [reply docs/book/stacks-and-components/component-guide/image-builders/kaniko.md:143]: Yes exactly because the options are only needed for specific cases

**Review by strickvl — APPROVED** [review]
Approved pending tests passing.

**Pull request merged**